diagnostic.md

```diff
@@ -55,6 +68,21 @@ How would we reference each of the following categories of elements?
 
 <!-- your answer starts here -->
 
+```CSS
+.big{
+
+}
+#contentPane {
+
+}
+.important .red {
```

Comment:
A space is a descendent relationship. In order to only apply styles to elements that have BOTH classes, you have to combine them. `.important.red`